features/achordion.c

```diff
@@ -211,6 +211,21 @@ bool process_achordion(uint16_t keycode, keyrecord_t* record) {
       tap_hold_record.event.pressed = false;
       // Plumb tap release event.
       recursively_process_record(&tap_hold_record, STATE_TAPPING);
+#ifdef ACHORDION_STREAK
+      streak_timer = record->event.time | 1;
+      if (is_streak && is_key_event && is_tap_hold && record->tap.count == 0) {
+        // If we are in a streak and resolved the current tap-hold key as a tap
+        // consider the next tap-hold key as active to be resolved next.
```

Comment:
Thanks for the clarifying comment. It seems debatable whether the next tap-hold key ought to be immediately settled as tapped in this case, but this behavior does seem like an easier fit to Achordion's implementation. WDYT?

Reply:
I didn't want to do that, because the next tap-hold might be shift, which (in my configuration) I leave exempted from the tap-hold logic. Having this just counts the tap-hold that just got settled as a tap in the logic, then we can monitor as normal going forward.

Reply:
Aha, right. Agreed the key should be unsettled to support such use.